experiment/autobumper/bumper/bumper.go

```diff
@@ -261,3 +534,37 @@ func generatePRBody(images map[string]string, assignment string) string {
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
 	return prowSummary + "\n\n" + testImagesSummary + "\n\n" + assignment + "\n"
 }
+
+func getAssignment(oncallAddress string) string {
+	if oncallAddress == "" {
+		return ""
+	}
+	oncaller, err := getOncaller(oncallAddress)
+	if err == nil {
+		if oncaller != "" {
+			return "/cc @" + oncaller
+		}
+		return "Nobody is currently oncall, so falling back to Blunderbuss."
+	}
+	return fmt.Sprintf("An error occurred while finding an assignee: `%s`.\nFalling back to Blunderbuss.", err)
+}
+
+func getOncaller(oncallAddress string) (string, error) {
+	req, err := http.Get(oncallAddress)
+	if err != nil {
+		return "", err
+	}
+	defer req.Body.Close()
+	if req.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("HTTP error %d (%q) fetching current oncaller", req.StatusCode, req.Status)
```

Comment:
IMO "fetching current oncaller" is unnecessary as it is trivial for the caller to figure this out, I'd do something more like:

`fmt.Errorf("get: HTTP error %d: %q", req.StatusCode, req.Status)`

Reply:
Makes sense. Changed as suggested.